dashboard/v1.1/src/layouts/Sidebar.tsx

```diff
@@ -32,7 +32,7 @@ const useStyles = makeStyles(theme => ({
   // Nested lists
   nested: {
     paddingLeft: theme.spacing(4),
-    backgroundColor: '#DCDCDC'
+    backgroundColor: 'white'
```

Comment:
```suggestion
    // backgroundColor: '#DCDCDC'
```

The white colour leads to an inconsistency in the dark mode.
<img width="243" alt="Screenshot 2021-03-22 at 8 46 48 AM" src="https://user-images.githubusercontent.com/36980003/111936233-4ab11900-8aeb-11eb-956c-99a27332e21b.png">

Reply:
Thanks for the suggested improvements. I am doing the prescribed changes and will make a PR ASAP.

Reply:
@Tushar3099 I think it will be better if you can update this PR itself.

Reply:
Yeah sorry, I meant the same :+1: . I will update this PR itself.